index.html

```diff
@@ -23,5 +24,152 @@ <h1 class="playfairFont">José Fernández-Caminero</h1>
   </main>
 
   <section class="middleDeco"></section>
+
+  <section>
+    <h2 class="playfairFont padding10twenty">Projects</h2>
+
```

Comment:
- [ ] I think it would be better if you add the correct spacing just above your `Projects` section to make your design sticks as much as possible to the figma design
![Screenshot from 2023-03-24 16-56-41](https://user-images.githubusercontent.com/83042742/227639279-41bd85bb-ba44-449c-8157-580dae66d7be.png)